universe/node/api.go

```diff
@@ -152,6 +148,12 @@ func (n *Node) RegisterAPI(r *gin.Engine) {
 				object.GET("/attributes/sub", n.apiGetObjectAttributeSubValue)
 
 				object.GET("/all-users/attributes", n.apiGetObjectAllUsersAttributeValuesList)
+
+				timeline := object.Group("/timeline")
+				{
+					timeline.GET("", n.apiTimelineForObject)
+					timeline.POST("", n.apiTimelineAddForObject)
```

Comment:
suspect the POST will need some additional permission checking wrapper around it